event-exporter/sinks/stackdriver/log_entry_factory.go

```diff
@@ -36,8 +35,6 @@ var (
 	fieldBlacklist = []string{
 		// Is unnecessary, because it's demuxed already
 		"count",
-		// Timestamp is in the logEntry's metadata
-		"lastTimestamp",
```

Comment:
Review note: Now that we might use any of lastTimestamp, lastObservedTimestamp, eventTime, or current time as log entry timestamp, I don't see the need to exclude lastTimestamp from log payload. We can probably dynamically add the field that we use to set log entry timestamp to this list but the logic will be more complicated than what we are doing today. Also I don't see the need to exclude the field just because we use it to set log entry timestamp. Therefore I removed this field from the list in this commit. Feel free to let me know if you have any concerns/suggestions!

Note that this will require a note in GKE release note since it is a user visible change.

Reply:
we'll also have to be transparent about this new behaviour of what the "timestamp" is in the release. Otherwise, exposing this field sgtm.

Reply:
Yes agreed. I think this PR needs a release notes whether we expose this field or not because it is a user visible change.